source/CardGames.py

```diff
@@ -636,7 +645,37 @@ def fugitive_repeating_turn(self):
                       #Functions to check for illegally placed hideouts and burns would go here
       elif turn == "pass":
         print("\nPassing...")
-      return NewHideouts
+    
+    def marshall_repeating_turn(self):
+      #Obtain user's deck choice
+      while True:
+        try:
+          deck = int(input("\nSelect which deck to draw two cards from ['1' (1-14), '2' (15-28), '3' (29-42)] and press enter... "))
+        except ValueError:
+          print("Invalid input, try again.")
+          continue
+        if deck in [1, 2, 3]:
+          break
+      
+      match deck:
+        case 1:
+          deck = self.LowRangeDeck
+        case 2:
+          deck = self.MidRangeDeck
+        case 3: 
+          deck = self.HighRangeDeck
+      
+      #Draw a card from selected deck
+      self.marshall.addCard(deck.pop(), isKnown=True)
+
+      #Show marshall's hand, including drawn card
+      print("\nThis is your hand:")
+      self.marshall.showHand()
```

Comment:
I am glad you are showing the hand and board after each turn! Good touch brother!

Reply:
Thank you brotha! Glad you liked the polishing.